<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriorityDialog</class>
 <widget class="QDialog" name="PriorityDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Label" name="cTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>8</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="cExit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority2">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority3">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority4">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority5">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority7">
   <property name="geometry">
    <rect>
     <x>212</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority8">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cPriority9">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>46</y>
     <width>27</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="PriorityLabel" name="cNone">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>275</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>widget/label.h</header>
  </customwidget>
  <customwidget>
   <class>PriorityLabel</class>
   <extends>QLabel</extends>
   <header>widget/prioritylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
